projects/ad9656_fmc/src/app/ad9508.c

```diff
@@ -0,0 +1,191 @@
+/***************************************************************************//**
+ * @file ad9508.c
+ * @brief Implementation of AD9508 Driver.
+ * @author DHotolea (dan.hotoleanu@analog.com)
+ ********************************************************************************
+ * Copyright 2020(c) Analog Devices, Inc.
+ *
+ * All rights reserved.
```

Comment:
looking better here aren't these the driver files ? You don't need to copy them here again. Just use the ones from drivers folder.

Reply:
These drivers are new, they are written by me.